website/docs/language/resources/ephemeral/write-only.mdx

```diff
@@ -78,7 +78,7 @@ resource "aws_db_instance" "example" {
 
 During a Terraform operation, the provider uses the `password_wo` value to create the database instance, and then Terraform discards that value without storing it in the plan or state file. 
 
-Note that the way this is written, the `password_wo` value is lost after Terraform generates unless we capture it in another resource or output. For example of generating, storing, retrieving, and using an ephemeral password as a write-only argument, refer to the [expanded example below](#example). 
+Note that the way this is written, the `password_wo` value is lost after Terraform generates unless we capture it in another resource or output. For example of generating, storing, retrieving, and using an ephemeral password as a write-only argument, refer to the [expanded example below](#examples). 
```

Comment:
```suggestion
Note that Terraform does not store the generated value for `password_wo`, but you can capture it in another resource or output. For an example of generating, storing, retrieving, and using an ephemeral password as a write-only argument, refer to [Examples](#examples). 
```
Replace "the way this is written" with more concrete language.

Reply:
Does the example actually show how to do this? I thought Brian's comment in our team chat was that there's no way to capture it for Azure? What do we advise users to do instead - would they put it in their own password store manually?

Reply:
The example does this now, so resolving this!